### ISSUE TITLE
TFC Copper does not integrate into Forge tags despite seemingly having a forge tag

### ISSUE THREAD

**Issue by FabioTheTurtle** [TerraFirmaCraft/TerraFirmaCraft#2232]
Among the copper ingot metadata is the tag "tag:items:forge:ingots:copper", which should allow it to be accessed by other mods as a generic copper ingot, which doesn't seem to work.

Brass, Zinc and Gold have these similar respective tags and DO allow cross mod compatibility.

**Comment by FabioTheTurtle** [comment]
Please add labels for 1.18.X

**Comment by alcatrazEscapee** [comment]
This issue report makes no sense. You're reporting that TFC copper both does and does not have a specific tag. I can resolve the confusion, it does. So rather than reporting something that is plainly impossible; what *actual thing* are you expecting to happen, what *actual problem* are you running into with copper ingots specifically. In other words, please define "which doesn't seem to work.".

As is; the issue is resolved by it's own description; copper ingots do tag themselves as `forge:ingots/copper`

https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/894c51cc94e983ef9b2d62578c49873d6a8790d3/src/main/resources/data/forge/tags/items/ingots/copper.json#L5

**Comment by FabioTheTurtle** [comment]
> In other words, please define "which doesn't seem to work.".

The problem is that despite the fact that TFC's copper ingots containing the relevant forge tags to allow cross mod compatibility, the "ore dictionary," means that it should be able to be cross compatible and freely used among other mods in crafting recipes, for example (assuming they've been coded to allow universal forge items).
This doesn't work, despite having the tag. The copper ingots are not being accepted by other mods as copper ingots.

What makes me consider that this is a bug, is that, as mentioned in the original post, TFC's Brass, Zinc and Gold also carry the universal forge tags for themselves and as such are able to be freely used across mods.

Logic dictates that the copper ingot should also share this function, but for some reason it is not.

**Comment by alcatrazEscapee** [comment]
You have still failed to articulate an actual problem and are just talking nonsense at this point. Please make an example of a **single actual crafting recipe** which you would expect to work with a **single actual other mod** that doesn't when you use this mod with TFC.

**Comment by FabioTheTurtle** [comment]
I've clearly explained what's happening, but if you insist I give a full example, then fine. Instead of just saying that "copper can't be used in other mods" I figured I'd give a technical example.

TFC copper is not available to use in Create recipes; it will not recognise them in crafting recipes, such as a pipe, and it won't allow it to be pressed in a machine.
Once again, this is despite the fact that they have the forge tags.

TFC's zinc can be broken into Create's zinc nuggets, and TFC brass can be applied to stripped logs to make casings.
TFC Zinc, Brass and Gold will show in valid crafting recipes when looked up in JEI, but Copper does not.

This image shows Copper containing the forge tags.
![2022-11-16_07 58 03](https://user-images.githubusercontent.com/118239332/202035338-8ecdd8b8-4bce-49cc-a6b5-78121b5a5bf6.png)

These images show that both coppers do not share the same crafting recipes.
![2022-11-16_07 58 39](https://user-images.githubusercontent.com/118239332/202035424-b3a33b33-a023-485b-b149-f29247cc9b61.png)
![2022-11-16_07 59 02](https://user-images.githubusercontent.com/118239332/202035429-e36bf890-6f99-4fdc-b884-7f7f080c932c.png)

These images show Brass containing the forge tags, and the fact that TFC Brass can be used in recipes.
![2022-11-16_08 02 24](https://user-images.githubusercontent.com/118239332/202035543-d367a348-35e9-4437-a904-59bb8aa379c8.png)
![2022-11-16_08 02 41](https://user-images.githubusercontent.com/118239332/202035549-18cd9384-29c1-495e-89c0-ed587f72b995.png)

These images show Zinc containing the forge tags, and the fact that TFC Zinc can be used in recipes.
![2022-11-16_08 03 33](https://user-images.githubusercontent.com/118239332/202035737-4a118aee-5ac4-4ae4-8fb2-94446df61528.png)
![2022-11-16_08 03 55](https://user-images.githubusercontent.com/118239332/202035757-52043120-efe3-40d2-bf9d-20d7187c71ee.png)

These images show Gold containing the forge tags, and the fact that TFC Gold can be used in recipes.
![2022-11-16_08 04 21](https://user-images.githubusercontent.com/118239332/202035809-acbce4a1-22ef-4fe5-8af6-b71d87ad3c98.png)
![2022-11-16_08 04 38](https://user-images.githubusercontent.com/118239332/202035821-28398923-8c9c-473c-b8e1-cf374da58789.png)

**Comment by alcatrazEscapee** [comment]
The tag exists, yes. That is as much as we can do.

> TFC copper is not available to use in Create recipes; it will not recognise [sic] them in crafting recipes, such as a pipe, and it won't allow it to be pressed in a machine.

Create **does not use the `forge:ingots/copper` tag**. It doesn't matter in the slightest what we do or don't tag it as. The create fluid pipe recipe you show, as an example, [requires specifically `minecraft:copper`](https://github.com/Creators-of-Create/Create/blob/cf87508276f845675e64ad88cbf013298aac72a3/src/generated/resources/data/create/recipes/crafting/kinetics/fluid_pipe.json#L11). Or [here](https://github.com/Creators-of-Create/Create/blob/cf87508276f845675e64ad88cbf013298aac72a3/src/generated/resources/data/create/recipes/crafting/materials/copper_nugget.json#L5). As opposed to where they [accept any `forge:ingots/zinc`](https://github.com/Creators-of-Create/Create/blob/cf87508276f845675e64ad88cbf013298aac72a3/src/generated/resources/data/create/recipes/crafting/materials/zinc_nugget_from_decompacting.json#L5)

In other words; I've been asking for a concrete example because you are failing to articulate an actual issue with TFC. We already do use the correct ingot tags, there is nothing wrong with their implementation from our end.

This is already filed as an issue with create (https://github.com/Creators-of-Create/Create/issues/3959) and will be entirely on 
them to fix.

**Comment by FabioTheTurtle** [comment]
Understood.
Thank you, apologies for dragging out this report.
